docker-compose.yml

```diff
@@ -1,8 +1,10 @@
-version: '3.7'
+version: "3.7"
 
 services:
   db:
     image: postgres:10.1-alpine
+    ports:
+      - 5432:5432
```

Comment:
Can we please remove this change? If you have a database running, this port forwarding setting conflicts and crashes the docker-compose invocation.

```
ERROR: for db  Cannot start service db: driver failed programming external connectivity on endpoint crt-portal_db_1 (2a0524dff68b3d8a0b69886e70b8090684d6876597db2a4ec90bc4a936ee06f0): Error starting userland proxy: listen tcp 0.0.0.0:5432: bind: address already in use
```